credentials.conf.template

```diff
@@ -32,3 +32,23 @@ TOMCAT_UN="tomcat"
 TOMCAT_GROUP="tomcat"
 
 SERVICES_PATH="/opt"
+
+# Use AWS SQS to deliver monitoring statuses, leave blank for omit
+MONITOR_AWS_SQS_ACCESS_KEY="[placeholder]"
+MONITOR_AWS_SQS_SECRET_KEY="[placeholder]"
+MONITOR_AWS_SQS_QUEUE_URL="[placeholder]"
+
+# Email addresses to deliver monitoring warnings and errors, leave blank for omit
```

Comment:
#EMail settings to deliver monitoring events via SMTP. Separate mailboxes may be configured for error and warning event types. Optional.

Reply:
Done.